packages/core/src/components/table/_table.scss

```diff
@@ -75,6 +75,26 @@ $dark-table-border-color: $pt-dark-divider-white !default;
       box-shadow: inset 0 $table-border-width 0 0 $table-border-color;
     }
   }
+
+  // a bunch of deep compound selectors ahead, but there's not really a better way to do this right now
+  // stylelint-disable selector-max-compound-selectors
+  .#{$ns}-dark & {
```

Comment:
there are now two dark style blocks in this file. can we combine them? or is there a particular reason?

Reply:
@giladgray it's a matter of whether it's in running text or not (for Markdown support). Before this PR, we had the bare minimum in the Sass placeholder in order to support Markdown documents. This PR adds the bare dark minimum for the Sass placeholder, so that we have bare minimum dark table styles in Markdown documents. I left everything else in the main selector, because it's not needed for Markdown docs